CMakeLists.txt

```diff
@@ -644,9 +644,14 @@ if (BUILD_CLIENT OR BUILD_TTY_CLIENT OR BUILD_SERVER OR BUILD_DUMMY_APP)
         else ()
             add_definitions(-DUSE_CURSES)
             set(CURSES_NEED_NCURSES 1) # Tells FindCurses that ncurses is required
-            find_package(CursesW REQUIRED)
-            set(LIBS_ENGINE_BASE ${LIBS_ENGINE_BASE} ${CURSESW_LIBRARIES})
-            include_directories(${CURSESW_INCLUDE_DIR})
+            find_package(CursesW)
+            if (CursesW_FOUND)
+                include_directories(${CURSESW_INCLUDE_DIR})
```

Comment:
The update to LIBS_ENGINE_BASE is missing in the first branch and there's no include_directories in the second branch.